mypy/checkmember.py

```diff
@@ -778,32 +783,39 @@ def analyze_class_attribute_access(itype: Instance,
         if not mx.is_lvalue:
             result = analyze_descriptor_access(mx.original_type, result, mx.builtin_type,
                                                mx.msg, mx.context, chk=mx.chk)
-        return result
+
+        return apply_class_attr_hook(mx, hook, result)
     elif isinstance(node.node, Var):
+        # TODO RIGHT NOW NOMERGE - is it okay to not modify this?
```

Comment:
I had a few questions in here (marked by these `TODO RIGHT NOW NOMERGE` comments/asserts), both around whether certain return values should be modified and also how to exercise some of these branches, would love some advice on both! 🙂